models/user/user.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+mongoose.Promise = global.Promise;
+const validator = require('validator');
+const mongodbErrorHandler = require('mongoose-mongodb-errors');
+const passportLocalMongoose = require('passport-local-mongoose');
+
+// Passport-Local Mongoose will add a username,
```

Comment:
make it a multi line comment please `/* */`

Reply:
Done